projects/soft_timer/src/main.c

```diff
@@ -0,0 +1,42 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+// Values are in milliseconds.
+#define COUNTER_A_DELAY 500
```

Comment:
It would be good to add a suffix to this to indicate the units (ie COUNTER_A_DELAY_MS)